operators/src/processing/raster_vector_join/non_aggregated.rs

```diff
@@ -256,6 +263,7 @@ where
 
     fn into_collection(self, new_column_name: &str) -> Result<FeatureCollection<G>> {
         let Some(state) = self.state else {
+            log::warn!("Empty input for vector raster join");
```

Comment:
Do we need an extra warning when we throw an error?

Reply:
no its a debug leftover. Will remove it later.

Reply:
[68e415f](https://github.com/geo-engine/geoengine/pull/824/commits/68e415f0df584517e614e3d889c52ce6f727c373)